<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix rows="3" columns="2" tab_name="plot">
   <plot style="Lines" col="0" row="0">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint1/velocity" G="119" custom_transform="noTransform" B="180" R="31"/>
    <curve name="/panda_joint_velocity_controller/command/data.0" G="39" custom_transform="noTransform" B="40" R="214"/>
    <transform value="noTransform"/>
   </plot>
   <plot style="Lines" col="0" row="1">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint3/velocity" G="76" custom_transform="noTransform" B="193" R="241"/>
    <curve name="/panda_joint_velocity_controller/command/data.2" G="103" custom_transform="noTransform" B="189" R="148"/>
    <transform value="noTransform"/>
   </plot>
   <plot style="Lines" col="0" row="2">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint5/velocity" G="119" custom_transform="noTransform" B="180" R="31"/>
    <curve name="/panda_joint_velocity_controller/command/data.4" G="39" custom_transform="noTransform" B="40" R="214"/>
    <transform value="noTransform"/>
   </plot>
   <plot style="Lines" col="1" row="0">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint2/velocity" G="201" custom_transform="noTransform" B="56" R="26"/>
    <curve name="/panda_joint_velocity_controller/command/data.1" G="127" custom_transform="noTransform" B="14" R="255"/>
    <transform value="noTransform"/>
   </plot>
   <plot style="Lines" col="1" row="1">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint4/velocity" G="190" custom_transform="noTransform" B="207" R="23"/>
    <curve name="/panda_joint_velocity_controller/command/data.3" G="189" custom_transform="noTransform" B="34" R="188"/>
    <transform value="noTransform"/>
   </plot>
   <plot style="Lines" col="1" row="2">
    <range bottom="-2.000000" right="106.645975" left="98.441222" top="2.000000"/>
    <limitY min="-2" max="2"/>
    <curve name="/joint_states/panda_joint6/velocity" G="201" custom_transform="noTransform" B="56" R="26"/>
    <curve name="/panda_joint_velocity_controller/command/data.5" G="127" custom_transform="noTransform" B="14" R="255"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="ROS /rosout Visualization" status="idle"/>
  <plugin ID="ROS Topic Re-Publisher" status="idle"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

